awx/main/access.py

```diff
@@ -1405,29 +1402,27 @@ def can_start(self, obj, validate_license=True):
         if self.user.is_superuser:
             return True
 
+        credential_access = all([self.user in cred.use_role for cred in obj.credentials.all()])
         inventory_access = obj.inventory and self.user in obj.inventory.use_role
-        credential_access = obj.credential and self.user in obj.credential.use_role
-        job_extra_credentials = set(obj.extra_credentials.all())
-        if job_extra_credentials:
-            credential_access = False
+        job_credentials = set(obj.credentials.all())
 
         # Check if JT execute access (and related prompts) is sufficient
         if obj.job_template is not None:
             prompts_access = True
             job_fields = {}
-            jt_extra_credentials = set(obj.job_template.extra_credentials.all())
+            jt_credentials = set(obj.job_template.credentials.all())
```

Comment:
@AlanCoding this basically works the same way as `.extra_credentials`, but is just extended to the entire `.credentials` relation now.